src/components/molecules/ZopaFooter/styles/Heading.tsx

```diff
@@ -1,6 +1,10 @@
 import styled from 'styled-components';
-import ZopaHeading from '../../../atoms/Heading/Heading';
+import Text from '../../../atoms/Text/Text';
+import { colors } from '../../../../constants';
 
-export const Heading = styled(ZopaHeading).attrs({ as: 'h3', size: 'h6' })`
+export const Heading = styled(Text).attrs({
+  weight: 'bold',
+  color: colors.greyDarkest,
+})`
   text-transform: uppercase;
```

Comment:
There's a `capitalize` prop for the `Text` that we can use here instead of the css

Reply:
The capitalised prop has a relationship with the size of the text, hence using that prop would mean that the text can only be small.